lib/vehicle_factory.rb

```diff
@@ -0,0 +1,27 @@
+class VehicleFactory
+              
+  def initialize
+  end
+
+  def create_vehicles(vehicle_list)
+    vehicle_list.map do |vehicle_data|
+      Vehicle.new(vehicle_data)
+    end
+  end
+
+  def rename_vehicles(vehicle_list)
```

Comment:
I think I might call this method something like "map_vehicle_attributes". You are literally taking attributes from some source and remapping them to match OUR source (the `Vehicle` class attributes). Naming aside, I really like that you've broken this out to its own method.